CONTRIBUTING.md

```diff
@@ -41,84 +38,99 @@ Before submitting a bug, try to recreate it and document the steps we can take t
+1. If your bug has already been submitted, feel free to comment and provide additional context (if different than the original submission).
+1. If your proposed fix is not in the open issues backlog, create an [issue](https://github.com/uswds/uswds/issues/new/choose) for the change you’re proposing. This helps us track our work.
+1. Follow the steps in the [Getting started with USWDS code](#getting-started-with-uswds-code) section above to get setup locally.
+1. Create a branch from `develop` and name it in a way that lightly defines what you’re working on (for example, `add-styles`).
+1. Once you’re ready to submit a pull request, fill out the [pull request template](https://github.com/uswds/uswds/compare).
+1. Link your pull request to the issue you created. This important step helps us know which issue this solution fixes. Tip: You can link the pull request in the body of the pull request template using the GitHub comment `closes #issue-no` or `resolves #issue-no`. You can read more about linking pull requests on [GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
+1. Submit your pull request against the `develop` branch.
 
+If the pull request is accepted, we will schedule the issue and merge the pull request for you.
+
+#### Submitting a pull request for a feature request or enhancement:
+
+1. Check our [open issues backlog](https://github.com/uswds/uswds/issues) for any duplicate or similar issues.
+2. If your idea has already been suggested, upvote that feature request with a thumbs up emoji (👍) and comment on the issue to let us know why you need this feature request or enhancement and any other supporting information.
+Tip: If you want to find other feature requests open for upvoting, check out our [feature requests sorted by upvotes](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Status%3A+Voting+Open+%F0%9F%91%8D%22+sort%3Areactions-%2B1-desc).
+3. If your proposed fix is not in the [open issues backlog](https://github.com/uswds/uswds/issues), [create an issue](https://github.com/uswds/uswds/issues/new?assignees=&labels=Type%3A+Feature+Request%2CStatus%3A+Triage&template=feature_request.yaml&title=USWDS+-+Feature%3A+%5BYOUR+TITLE%5D) describing your proposal. This doesn’t mean we don’t want you to create a pull request. We simply want to start the process with an online conversation first. Plus, other community members might have supporting thoughts to add to your proposal. If you’ve already got a pull request, no worries. Go ahead and attach it to the issue.
```

Comment:
**suggestion (non-blocking):** Should we switch all the ordered list to all use `1.` ? Or at least the lists affected by this work to reduce changed files.

Reply:
I think it's worth making this change for consistency as well